elixir-format.el

```diff
@@ -125,54 +125,76 @@ Shamelessly stolen from go-mode (https://github.com/dominikh/go-mode.el)"
               (error "Invalid rcs patch or internal error in elixir-format--apply-rcs-patch"))))))))
   )
 
+(defun elixir-mix-path-set? ()
+  (and (not (= (length elixir-format-mix-path) 0)) (not (= (length elixir-format-elixir-path) 0)))
+  )
+
+(defun display-configuration-missing-error ()
+  (let ((errbuff (get-buffer-create "*elixir-format-configuration-missing-errors*")))
```

Comment:
It's not clear to me why we create another error buffer. Why can't we reuse the other one?

Reply:
done.